pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "rapidpro-python"
+version = "2.8.0"
```

Comment:
for bonus points should get `fab2 rev` working with python libraries (dash, smartmin etc) but for now still need to do this manually. Update this version and the changelog, run `poetry publish`